packages/atomic/cypress/integration/search-box.cypress.ts

```diff
@@ -57,11 +60,13 @@ describe('Search Box Test Suites', () => {
       actionCause: 'searchboxSubmit',
     };
 
-    const fetchAnalytic = await getUAFetch('@coveoAnalytics');
+    const fetchAnalytic = await getAnalyticsSearchEventRequest(
```

Comment:
In master I uncommented the line below and it needs access to the response as well.